### PR TITLE
temporary-fix(big bundle & cannot-resolve-symbol): revert to non-optimized bundling until it's fixed in qwik

### PR DESCRIPTION
**Pull request by maiieul**
**TLDR**:  I think it would be best to keep the unoptimized bundle until a real fix can be made in the optimizer or until we're ready for v1 if the optimizer issue hasn't been fixed by then.

# What is it?

- [ ] Feature / enhancement
- [x] Bug
- [ ] Docs / tests

# Description

I removed

```tsx
      output: {
        preserveModules: true,
        preserveModulesRoot: 'packages/kit-headless/src',
      },
```

from the headless-kit's vite config, because it creates the namespace issues described in #519, for which there is no great workaround in terms of DX.

This brings back the big bundle issue, but the "big bundle" isn't so big on first load anyway, and subsequent requests will be loaded from the browser cache.

To make the bundle smaller, I removed country-list-json and libphonenumber-js dependencies and all related code (which didn't receive any attention for the past 6 months). We can always add it back later or create another package for those kinds of components if the bug hasn't been fixed by then.

Performance should be okay now (11kb on first load :point_down:)

image

:warning:  I reused some of the code from #511, updated to latest qwik version, and made sure that everything works in dev and preview (the workspace dependency was causing issues to the build step). I suggest to first merge this PR and then @shairez you can work on top of it for the utils package.

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/qwikifiers/qwik-ui/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

**Comment by maiieul** [comment]
In case anyone's looking into this, I had a programming session with Shai and we think we know what's causing the namespace issues when using `preserveModules`, so we're holding off for now.

**Comment by shairez** [comment]
closing this in favor of the recent fixes
